File: app/src/main/java/com/rakuishi/ok/models/Gist.java
package com.rakuishi.ok.models;

import com.google.gson.annotations.SerializedName;

public class Gist {

    @SerializedName("id")
    public final String id;

    @SerializedName("description")
    public final String description;

    @SerializedName("updated_at")
    public final String updatedAt;

    @SerializedName("html_url")
    public final String url;

    public Gist(String id, String description, String updatedAt, String url) {
        this.id = id;
        this.description = description;
        this.updatedAt = updatedAt;
        this.url = url;
    }

    @Override
    public String toString() {
        return "id: " + id + "\n" +
               "description: " + description + "\n" +
               "updated_at: " + updatedAt + "\n" +
               "url: " + url;
    }
}

File: app/src/main/java/com/rakuishi/ok/data/Gist.kt
package com.rakuishi.ok.data

import com.google.gson.annotations.SerializedName

data class Gist(
    @SerializedName("id")
    val id: String,
    @SerializedName("description")
    val description: String,
    @SerializedName("updated_at")
    val updatedAt: String,
    @SerializedName("html_url")
    val url: String
)